### PR TITLE
Update documentation to use tox

### PR DESCRIPTION
**Pull request by jainvikas8**
### Description

 Advice on using Tox rather than pipenv

### Test Coverage

- [x ]  This change is covered by existing or additional automated tests.
- [ ]  Manual testing has been performed (and evidence provided) as automated testing was not feasible.
- [ ]  Additional tests are not required for this change (e.g. documentation update).

### CODE REVIEW

**Comment by jainvikas8** [comment]
Merge after #84

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -49,25 +49,17 @@ Clone the `mbed-tools` GitHub repository
...
 
-Setup Pipenv to use Python 3 and install package development dependencies:
+Set up the development environment using tox (tox will create a development environment at mbed-tools/.venv):
 
 ```bash
 cd mbed-tools/
-pipenv --three
-pipenv install --dev --skip-lock
+tox -e dev
+. .venv/bin/activate
```
Comment (Patater) [inline DEVELOPMENT.md:57]: would recommend `source` instead of just the dot, as new developers might think the dot to be a strange typo or something.

**Comment by jainvikas8** [comment]
Forced pushed to address comments from @Patater

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -49,25 +49,17 @@ Clone the `mbed-tools` GitHub repository
...
 
-Setup Pipenv to use Python 3 and install package development dependencies:
+Set up the development environment using tox (tox will create a development environment at mbed-tools/.venv):
 
 ```bash
 cd mbed-tools/
-pipenv --three
-pipenv install --dev --skip-lock
+tox -e dev
+<source_dir> .venv/bin/activate
```
Comment (Patater) [inline DEVELOPMENT.md:57]: No. `.` doesn't mean source dir. It's shorthand for the command `source`.

```bash
cd mbed-tools/
tox -e dev
source .venv/bin/activate
```

**Review by Patater — CHANGES_REQUESTED** [review]

**Pull request merged**